empty-script triggers warning when passing a function as argument

The new `empty-script` rule (nice addition btw) triggers warnings when passing a function as argument. According to the [protractor documentation](https://www.protractortest.org/#/api?view=webdriver.WebDriver.prototype.executeScript) the passed function "will be converted to a string" automatically.

On the technical side it happens because the node type is `Identifier` and not `Literal` and it does not hold the `.value` property thus failing the test.

## Steps to reproduce

```js
function myClientScript(){
    /* this gets executed in the browser */
}

browser.executeScript(myClientScript);
```
## Expected results

No errors or warnings.

## Actual results

    5:9  warning  executeScript() called with an empty script  protractor/empty-script

--- a/lib/rules/empty-script.js
+++ b/lib/rules/empty-script.js
@@ -5,6 +5,10 @@
  * @author Alexander Afanasyev
  */
 
+function isLiteral (node) {
+  return node.type === 'Literal'
+}
+
 module.exports = {
   meta: {
     schema: []
@@ -29,7 +33,12 @@ module.exports = {
               return
             }
 
-            var firstArgumentNonEmpty = argumentExists && node.arguments[0].value
+            var firstArgument = node.arguments[0]
+            if (!isLiteral(firstArgument)) {
+              return
+            }
+
+            var firstArgumentNonEmpty = firstArgument.value
             if (!firstArgumentNonEmpty) {
               context.report({
                 node: property,

--- a/test/rules/empty-script.js
+++ b/test/rules/empty-script.js
@@ -10,7 +10,9 @@ eslintTester.run('empty-script', rule, {
     'browser.executeScript("var a = 1;");',
     'browser.executeAsyncScript("var a = 1;");',
     'var tag = browser.executeScript("return arguments[0].tagName", el);',
-    'browser.executeAsyncScript("var callback = arguments[arguments.length - 1];");'
+    'browser.executeAsyncScript("var callback = arguments[arguments.length - 1];");',
+    'var script = "var a = 1"; browser.executeScript(clientScript);',
+    'function clientScript(){ return 1; }; browser.executeScript(clientScript);'
   ],
 
   invalid: [

